### PR TITLE
Release notes for 2.5.0

### PR DESCRIPTION
**Pull request by dileepds**

### CODE REVIEW

File: release-notes/v2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.5.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+|CON-648|csi.k8s|Implement topology in the CSI driver|CSI supports the concept of topology by providing the mechanism to define availability zones and associate cluster nodes to those zones.|
+|CON-2508|csi.k8s|Post reboot and drained worker node is not cleaned up properly, unless flush it out|Safely removing or cleaning the stale mount paths or devices attached to a worker node.|
+|CON-3071|csi.k8s|Node monitor support|Node monitor running on every cluster node make sure that the node is healthy.|
```
Comment (AnushaY1916) [inline release-notes/v2.5.0.md:17]: Making sure the node is healthy might be misleading, we can specify something like "make sure no stale multipath entries exist"

Comment (dileepds) [reply release-notes/v2.5.0.md]: Michael corrected the resolution note for this and those changes are merged here now.

File: release-notes/v2.5.0.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.5.0.
+
+| ID | Component | Title | Resolution |
+|----|-----------|-------|------------|
+| CON-648  | csi.k8s | Implement basic topology in the CSI driver | StorageClasses may now be decorated with topology requirements and use volumeBindingMode: WaitForFirstConsumer |
+| CON-2508 | csi.k8s | Post reboot and drained worker nodes are not cleaned up properly unless manually tidied up | Safely removing stale mount paths or devices attached to a compute node. |
+| CON-3071 | csi.k8s | CSI node monitor support | A node monitor is now patrolling the block devices to ensure they're healthy. |
+| CON-2137 | csi.k8s | Run filesystem check before mount | Checking for filesystem corruption and optionally repairi it with the fsRepair StorageClass parameter. |
```
Comment (AnushaY1916) [inline release-notes/v2.5.0.md:18]: repairi -- Spelling

Comment (dileepds) [reply release-notes/v2.5.0.md]: Corrected

**Review by datamattsson — APPROVED** [review]
LGTM

**Review by AnushaY1916 — APPROVED** [review]

**Pull request merged**